### PR TITLE
add unit tests

### PR DESCRIPTION
**Pull request by MariiaNebesnova**

### CODE REVIEW

File: src/js/blocks/autoFind/tests/autoFind.test.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+import { act } from "react-dom/test-utils";
+import { render, unmountComponentAtNode } from "react-dom";
+import * as pageDataHandlers from "../autoFindProvider/pageDataHandlers";
+import {
+  abovePerception,
+  generationData,
+  interactedGenerationData,
+  mockResponseData,
+  perceptiontreshold,
+  perceptiontresholdLow,
```
Comment (Bryant-24) [inline src/js/blocks/autoFind/tests/autoFind.test.js:13]: `perceptionTreshold` and `perceptionTresholdLow` I believe?

File: src/js/blocks/autoFind/tests/autoFind.test.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+  const TestComponent = () => {
+    const [
+      { status, predictedElements, pageElements, availableForGeneration },
+      { identifyElements, onChangePerception },
+    ] = useAutoFind();
+
+    return (
+      <React.Fragment>
+        <button id="idetify" onClick={identifyElements}></button>
```
Comment (Bryant-24) [inline src/js/blocks/autoFind/tests/autoFind.test.js:33]: `id="identify"`

File: src/loadPanel.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 window.onload = () => {
-  /*global chrome*/
-  /*eslint no-undef: "error"*/
+  /* global chrome*/
+  /* eslint no-undef: "error"*/
```
Comment (Bryant-24) [inline src/loadPanel.js:3]: `/* eslint no-undef: "error" */` - whitespace on both sides

**Review by Bryant-24 — APPROVED** [review]
PR looks good for me - thank you!

File: src/js/blocks/autoFind/tests/autoFind.test.js (hunk 1 above)
Comment (MariiaNebesnova) [reply src/js/blocks/autoFind/tests/autoFind.test.js:13]: yep, different values for different test cases

File: src/loadPanel.js (hunk 3 above)
Comment (MariiaNebesnova) [reply src/loadPanel.js]: sure

File: src/js/blocks/autoFind/tests/autoFind.test.js (hunk 2 above)
Comment (MariiaNebesnova) [reply src/js/blocks/autoFind/tests/autoFind.test.js]: thank you! fixed

**Pull request merged**